packages/bento-design-system/src/Charts/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Bar,
+  BarChart as RechartBarChart,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import { useBentoConfig } from "../../BentoConfigContext";
+import { bodyRecipe } from "../../Typography/Body/Body.css";
+import { allColors } from "../../util/atoms";
+import { vars } from "../../vars.css";
+import { ChartProps } from "../ChartProps";
+import { legendContent } from "../Legend/Legend";
+import { tooltipContent } from "../Tooltip/Tooltip";
+
+type Props<D extends string, C extends string> = ChartProps & {
+  data: Record<D | C, unknown>[];
+  categories: C[];
+  dataKey: D;
+  showXAxis?: boolean;
+  showYAxis?: boolean;
+  stacked?: boolean;
+};
+
+export type { Props as BarChartProps };
+
+export function BarChart<D extends string, C extends string>({
+  data,
+  dataKey,
+  categories,
+  showXAxis = true,
+  showYAxis = true,
+  showLegend = true,
+  showAnimation = true,
+  showTooltip = true,
```

Comment:
just a little obsession of mine: since we're defaulting these to `true`, wouldn't it be better to invert them and call them `hideXXX` instead, so to default them to `false`? In this way, the default is = to not passing any prop, and to disable them you just need to pass `hideXXX`, instead of `showXXX={false}`

Reply:
It's a good point, I've basically used the API as Tremor, but I agree it should be better to change the order